napalm_panos/panos.py

```diff
@@ -31,11 +31,12 @@
 
 from napalm_base.base import NetworkDriver
 from netmiko import ConnectHandler
+from netmiko import __version__ as netmiko_version
 
 
 class PANOSDriver(NetworkDriver):
 
-    def __init__(self, hostname, username, password, timeout=60, optional_args=None):
+    def __init__(self, hostname, username, password='', timeout=60, optional_args=None):
```

Comment:
Because if you omit the username and the password someone code write code agains `napalm-panos` that will not be compatible with other drivers. The entire point of `napalm` is to write vendor agnostic code, if we start making method signatures flexible depending on the driver we are defeating our own purpose.